packages/pnpm-sync/src/start.ts

```diff
@@ -16,7 +16,20 @@ program
       await pnpmSyncCopyAsync({
         getPackageIncludedFiles: PackageExtractor.getPackageIncludedFilesAsync,
         forEachAsyncWithConcurrency: Async.forEachAsync,
-        ensureFolder: FileSystem.ensureFolderAsync
+        ensureFolder: FileSystem.ensureFolderAsync,
+        logMessageCallback: (message: string, messageType: ILogMessageKind) => {
```

Comment:
This might be a good time to finish the [pnpm-sync-api-demo](https://github.com/tiktok/pnpm-sync/tree/main/pnpm-sync-api-demo) project. 

Actually calling your own API is the best way to find out if it's terrible or awesome. 😉

Reply:
Yeah, I will do it in a separated PR